core/node/da_dispatcher/src/metrics.rs

```diff
@@ -0,0 +1,29 @@
+use std::time::Duration;
+
+use vise::{Buckets, Gauge, Histogram, Metrics, Unit};
+
+#[derive(Debug, Metrics)]
+#[metrics(prefix = "server_da_dispatcher")]
+pub(super) struct DataAvailabilityDispatcherMetrics {
+    /// Latency of the dispatch of the blob.
+    #[metrics(buckets = Buckets::LATENCIES, unit = Unit::Seconds)]
```

Comment:
How large are these latencies supposed to be? `Buckets::LATENCIES` are good for low-ish latencies (you can look up the definition of this constant; most of the bucket thresholds are below 1s), so if the latency is supposed to be large, it could make sense to define buckets specifically for it.

Reply:
defined the buckets specifically for it